_summaries/he2019momentum_1.md

```diff
@@ -1,19 +1,87 @@
+Three variations of contrastive loss are visualized below along with their results using the linear classification protocol.
+
+![loss_variations](https://user-images.githubusercontent.com/7085644/140275575-0af2d328-263c-4c6d-a10e-97256fee5991.PNG)
+
+![contrastive_loss_variation_results](https://user-images.githubusercontent.com/7085644/140275560-d8fd836c-0938-47e9-8588-a46ae0898c64.PNG)
+
+All three methods show that larger K is beneficial, and therefore a larger dictionary is always better.
+Additionally, the results of using a larger momentum value are shown below and reveal that slow evolution is best.
+
+![momentum_results](https://user-images.githubusercontent.com/7085644/140275813-4f6cbc3d-fb3c-45f0-bcdd-3bf162a5041a.PNG)
+
+Comparison with previous unsupervised learning methods are done with accuracy vs. #parameters trade-offs.
+MoCo is able to achieve 60.6% accuracy compared to competitors of similar model sizes and also does best with large models as shown below.
+
+![lcp_results](https://user-images.githubusercontent.com/7085644/140276299-f4da7a64-cf85-4740-834b-517a80fa7e23.PNG)
+
+Unsupervised learning should learn features that are transferrable to downstream tasks.
+Feature normalization and schedules for fine-tuning MoCo are performed so that MoCo uses the same hyper-parameters and schedule as the supervised ImageNet counterpart.
+PASCAL VOC object detection was used to evaluate Moco vs. ImageNet supervised pre-training.
+The results below show that MoCo is able to beat the supervised learning counterpart for VOC object detection.
+
+![VOC_results](https://user-images.githubusercontent.com/7085644/140277865-a8416ec7-578b-46ad-9c57-9ab2b86f513a.PNG)
+
+And here are results showing MoCo beating the supervised counterpart with a C4 backbone.
+
+![VOC_results_2](https://user-images.githubusercontent.com/7085644/140277998-3b55cf93-a2f8-442e-96a3-18da9bf20fe0.PNG)
+
+Here are the results for the COCO dataset showing that MoCo is better than the supervised counterpart for both backbones.
+
+![COCO_results](https://user-images.githubusercontent.com/7085644/140278250-957a2f3a-652c-4551-8849-67da4a24acdd.PNG)
+
+Finally, downstream task evaluation reveals that MoCo is competitive with ImageNet supervised pre-training.
+
+![downstream_evaluation](https://user-images.githubusercontent.com/7085644/140278379-59d5c698-4c63-4d7b-84d9-c641b941d64c.PNG)
+
+The better results with MoCo pre-trained on the less curated dataset IG-1B compared to IN-1M shows that MoCo is good for real-world unsupervised learning.
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+* Unsupervised visual learning is challenging due to the continuous, high-dimensional space.
+* A dynamic dictionary that is large and consistent can be built using Momentum Contrast for unsupervised visual learning.
+* A queue of data samples for the dictionary allows for a large dictionary size while a momentum update allows for slow evolution and therefore consistency.
```

Comment:
Good summary. Went over the key points in the paper in an easy to understand format. I think if anything, it might be worth to add a few notes from the appendix, like A.8 and A.9 for talking about variations and ablations.